### PR TITLE
Documentation shoring

### PR DESCRIPTION
**Pull request by woodruffw**
* Properly mark constants as private.

* Fill in some missing method descriptions and correct
formatting errors.

* Prefer "@api private" to "@private" from here on out.

This is low priority, since it doesn't actually change anything in code. On the other hand, it finally gets us to 100% yardoc coverage :tada: 

cc @UniqMartin

### CODE REVIEW

File: lib/macho/utils.rb [hunk 1]
```diff
@@ -27,10 +30,11 @@ def self.specialize_format(format, endianness)
       format.tr("=", modifier)
     end
 
+    # Packs tagged strings into the an aligned payload.
```
Comment (UniqMartin) [inline lib/macho/utils.rb]: This sentence seems to be broken.

File: lib/macho/utils.rb [hunk 2]
```diff
@@ -47,36 +51,42 @@ def self.pack_strings(fixed_offset, alignment, strings = {})
...
     end
 
+    # Compares the given number to valid 64-bit Mach-O magic numbers.
     # @param num [Fixnum] the number being checked
     # @return [Boolean] true if `num` is a valid 64-bit magic number, false otherwise
     def self.magic64?(num)
       num == MH_MAGIC_64 || num == MH_CIGAM_64
     end
 
+    # Compares the given number of valid little-endian magic numbers.
```
Comment (UniqMartin) [inline lib/macho/utils.rb]: This sentence is slightly broken.

File: lib/macho/utils.rb [hunk 3]
```diff
@@ -47,36 +51,42 @@ def self.pack_strings(fixed_offset, alignment, strings = {})
...
     # @param num [Fixnum] the number being checked
     # @return [Boolean] true if `num` is a valid little-endian magic number, false otherwise
     def self.little_magic?(num)
       num == MH_CIGAM || num == MH_CIGAM_64
     end
 
+    # Compares the given number of valid big-endian magic numbers.
```
Comment (UniqMartin) [inline lib/macho/utils.rb]: This sentence is slightly broken.

**Comment by UniqMartin** [comment]
Thanks for keeping an eye on the documentation! I have to admit this hasn't been the most exciting diff. 😜 Thus I might have overlooked something.

:+1: Feel free to merge after fixing those minor mistakes I managed to spot nonetheless.

**Comment by woodruffw** [comment]
> I have to admit this hasn't been the most exciting diff

Yep, definitely a boring one :smile:. I'll merge in a few minutes...

**Comment by UniqMartin** [comment]
👏

**Pull request merged**